src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -1,179 +1,214 @@
+              }
+            </div>
+          </div>
+        ))}
+
+      <ul className="UserInformationContent__profile-links">
+        {user?.profileLinks?.map((profileLink) => (
+          <li key={profileLink.title}>
+            {profileLink.title}{" "}
+            <button
+              className="button--a"
+              onClick={editProfileLink(profileLink)}
+            >
+              Edit
+            </button>
+          </li>
+        ))}
+      </ul>
+
+      <button
+        className="UserInformationContent__add-profile-link button--a"
+        onClick={editProfileLink(undefined)}
+      >
+        Add a profile link
+      </button>
 
       {IS_BURN && (
         <>
           <label
             htmlFor="chk-kidsMode"
-            className={`checkbox ${profile?.kidsMode && "checkbox-checked"}`}
+            className={`checkbox ${user?.kidsMode && "checkbox-checked"}`}
           >
             Kids Mode
           </label>
           <input
             type="checkbox"
             name="kidsMode"
             id="chk-kidsMode"
-            defaultChecked={profile?.kidsMode || false}
+            defaultChecked={user?.kidsMode || false}
             onClick={() => toggleKidsMode()}
           />
           <label
             htmlFor={"chk-anonMode"}
-            className={`checkbox ${profile?.anonMode && "checkbox-checked"}`}
+            className={`checkbox ${user?.anonMode && "checkbox-checked"}`}
           >
             Anonymous Mode
           </label>
           <input
             type="checkbox"
             name={"anonMode"}
             id={"chk-anonMode"}
-            defaultChecked={profile?.anonMode || false}
+            defaultChecked={user?.anonMode || false}
             onClick={() => toggleAnonMode()}
           />
         </>
       )}
-
-      {venue?.showBadges && <Badges user={userWithId} currentVenue={venue} />}
-
+      {venue?.showBadges && <Badges user={user} currentVenue={venue} />}
       <label
         htmlFor="chk-mirrorVideo"
-        className={`checkbox ${profile?.mirrorVideo && "checkbox-checked"}`}
+        className={`checkbox ${user?.mirrorVideo && "checkbox-checked"}`}
       >
         Mirror my video
       </label>
-
       <input
         type="checkbox"
         name="mirrorVideo"
         id="chk-mirrorVideo"
-        defaultChecked={profile?.mirrorVideo || false}
+        defaultChecked={user?.mirrorVideo || false}
         onClick={() => toggleMirrorVideo()}
       />
-
-      <input
-        className="btn button btn-primary"
-        value="Edit profile"
-        onClick={() => setIsEditMode(true)}
-        type="button"
-      />
-
-      <input
-        className="btn button"
-        value="Change password"
-        type="button"
-        onClick={() => setIsPasswordEditMode(true)}
-      />
-
-      <input
-        className="btn button"
-        value="Log out"
-        onClick={logout}
-        type="button"
-      />
-    </>
```

Comment:
Where did we move these controls?

Reply:
The view is changed a bit. 

It looked this way:
![image](https://user-images.githubusercontent.com/63313732/123016223-1f62f200-d398-11eb-989a-1d9f447e119d.png)


This is the way it looks now:
![image](https://user-images.githubusercontent.com/63313732/123016292-46212880-d398-11eb-94bd-c4255ca6e262.png)

* edit input -> `editIcon` https://github.com/sparkletown/sparkle/blob/b40d96493471fd353df1a05041e18c3b7e9cbd00/src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx#L117:L122
* change password, logout -> https://github.com/sparkletown/sparkle/blob/b40d96493471fd353df1a05041e18c3b7e9cbd00/src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx#L203:L211